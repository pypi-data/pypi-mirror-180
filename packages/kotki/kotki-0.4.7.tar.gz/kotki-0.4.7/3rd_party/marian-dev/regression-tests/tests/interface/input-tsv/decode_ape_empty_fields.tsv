The files are stored in the Adobe / Fireworks CS4 folder 	Die Dateien werden in der Adobe / Fireworks CS4 Ordner
Right-click the document in Windows Explorer , choose Properties , and click the PDF tab .	
Also see function literal , and string literal .	
Then perform light touchup in Acrobat Pro or Acrobat Pro Extended after you add the fillable form fields .	Führen Sie dann die helle Retuschierung in Acrobat Pro oder Acrobat Pro Extended , nachdem Sie die ausfüllbaren Formularfelder hinzufügen .
	Halten Sie beim Ziehen des Zeigers über die Bewegung des Objekts horizontal , vertikal oder diagonal einzuschränken ( ein Vielfaches von 45 ° ) .
For web users to see your application , embed it in a web page .	Bei Webbenutzern verwendet wird , um die Anwendung anzuzeigen , die in einer Webseite eingebettet werden .

Display the Property inspector ( Window > Properties ) or panel to use :	Zeigen Sie den Eigenschafteninspektor ( " Fenster " " Eigenschaften " ) oder " verwenden , um :
Its behavior is similar to using the Free Transform command on a rectangle selection .	Das Verhalten ist ähnlich wie bei Verwendung des Befehls " Frei transformieren " auf ein Rechteck .
You can also perform many types of transformations by dragging the bounding box for a selection .	Sie können auch zahlreiche Transformationsarten durchführen , indem Sie den Begrenzungsrahmen für eine Auswahl .

	Zusammengesetzte Formen können einfache oder zusammengesetzte Pfade , Textrahmen , Textgliederung oder andere Formen aufgeholt werden .
	Setzen Sie die Nummer , um eine bestimmte Anzahl von der Nummerierung der Listenelemente beginnen soll .
The Gradient tool cannot be used with bitmap or indexed-color images .	Das Verlaufswerkzeug kann nicht mit Bitmap- oder indizierten Farben verwendet werden .
Choose an option from the Key Algorithm menu .	Wählen Sie eine Option aus dem Menü " Algorithm . "
Select Edit > Timeline > Copy Motion .	
	Passt die Größe eines Rahmens , um dessen Inhalt einzupassen .
If there are no compilation errors , you will see a Flash Player window showing the Hello World application .	Wenn keine Kompilierungsfehler vorliegen , wird eine Flash Player-Fenster mit der Anwendung " Hello World " angezeigt .
The policy file must load before you can extract data .	Die Richtliniendatei geladen werden müssen , bevor Sie Daten extrahieren können .
For additional shortcuts , see Keys for using the Animation panel in Timeline Mode ( Photoshop Extended ) .	Für weitere Tastenkombinationen finden Sie unter Tastenkombinationen zur Verwendung des Animationsbedienfelds in Zeitleiste Modus ( Photoshop Extended ) .
