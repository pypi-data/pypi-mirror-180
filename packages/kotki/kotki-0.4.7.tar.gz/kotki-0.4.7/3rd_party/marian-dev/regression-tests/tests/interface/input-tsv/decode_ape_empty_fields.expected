Die Dateien werden im Adobe / Fireworks CS4 Ordner gespeichert .
Klicken Sie mit der rechten Maustaste auf das Dokument in Windows Explorer , wählen Sie Eigenschaften aus , und klicken Sie auf die Registerkarte PDF .
Ebenfalls siehe Funktion literal , und String literal .
Führen Sie dann die helle Retuschierung in Acrobat Pro oder Acrobat Pro Extended durch , nachdem Sie die ausfüllbaren Formularfelder hinzugefügt haben .

Bei Webbenutzern , um die Anwendung anzuzeigen , die in einer Webseite eingebettet ist .

Zeigen Sie den Eigenschafteninspektor ( " Fenster " > " Eigenschaften " ) oder das Bedienfeld an :
Das Verhalten ist ähnlich wie bei Verwendung des Befehls " Frei transformieren " über eine Rechteckauswahl .
Sie können auch zahlreiche Transformationsarten durchführen , indem Sie den Begrenzungsrahmen für eine Auswahl ziehen .



Das Verlaufswerkzeug kann nicht mit Bitmap- oder indizierten Farben verwendet werden .
Wählen Sie eine Option aus dem Menü " Key Algorithm . "
Wählen Sie " Bearbeiten " > " Zeitleiste " > " Bewegung kopieren . "

Wenn keine Kompilierungsfehler vorliegen , wird ein Flash Player-Fenster mit der Anwendung " Hello World " angezeigt .
Die Richtliniendatei muss geladen werden , bevor Sie Daten extrahieren können .
Weitere Tastenkombinationen finden Sie unter " Tastenkombinationen " zur Verwendung des Animationsbedienfelds im Zeitleistenmodus ( Photoshop Extended ) .
